utils/market.ts

```diff
@@ -204,3 +204,80 @@ export async function getPoolIdsInfo(tokenIds: BN[]) {
   }
   return { swapPoolList, firstToken, lastToken };
 }
+
+export async function getPoolsForTrading() {
```

Comment:
Add rpc to the  begining of the function name, so its easier to find it later, and move it to Market static class.

Reply:
Done